update_version_next.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+"""
+Replace `.. versionchanged:: next` lines in docs files by the given version.
+
+Run this at release time to replace `next` with the just-released version
+in the sources.
+
+No backups are made; add/commit to Git before running the script.
+
+Applies to all the VersionChange directives. For deprecated-removed, only
+handle the first argument (deprecation version, not the removal version).
+
+"""
+
+import argparse
+import re
+import sys
+from pathlib import Path
+
+DIRECTIVE_RE = re.compile(
+    r"""
+        (?P<before>
+            \s*\.\.\s+
+            (version(added|changed|removed)|deprecated(-removed)?)
+            \s*::\s*
+        )
+        next
+        (?P<after>
+            .*
+        )
+    """,
+    re.VERBOSE | re.DOTALL,
+)
+
+doc_dir = Path(__file__).parent.parent.resolve()  # cpython/Doc/tools  # cpython/Doc
```

Comment:
Black has jumbled up these comments:

```python
doc_dir = (Path(__file__)
           .parent  # cpython/Doc/tools
           .parent  # cpython/Doc
           .resolve()
           )
```

[`c482123` (#164)](https://github.com/python/release-tools/pull/164/commits/c482123fe488581ea1f8817935a02eeaf353cda8#diff-ae24a10c93bce91291f30c5c206ef14058dc8d4f8dc73c36cbf9770112a8e1ddL35)

This worked if this tool was in the CPython repo (where it was originally, in https://github.com/python/cpython/pull/121278) but maybe we don't want a default here?

`run_release.py` doesn't have a default for `--repository REPO    Location of the CPython repository` and in practice builds I've been calling it as `--repository ../cpython` but it could be anywhere.